### ISSUE TITLE
Display capo fret

### ISSUE THREAD

**Issue by BigHonkinDaddy** [powertab/powertabeditor#325]
The "Mixer" pop-up allows for setting the capo fret.  It'd be great to see this this value above the applicable Tab staff, near the player name.  (I believe at this time, Power Tab Editor only allows a single capo fret setting for a given instrument for the entire score.)

**Comment by cameronwhite** [comment]
The capo is per-player (it's part of the tuning) so this might be the same request as #324 ?

**Comment by BigHonkinDaddy** [comment]
Thanks...yeah, was thinking that and even originally just submitted one github issue for both, but then I realized that the tuning would be the same for the whole score (for a player/instrument) but as I alluded above, the capo could change during the score so thought it might be a separate enhancement.

**Comment by cameronwhite** [comment]
Yeah, I think this would then be a request to be able to change the capo fret for a player at the same time as e.g. changing which instrument it's assigned to.

**Comment by NalihznehC** [comment]
In the current case,
![image](https://user-images.githubusercontent.com/39670573/192881128-adddc2f7-1fce-457c-98c7-ac7eaa8c49da.png)
if I am going to change the capo in the future, I have to create a new player:
![image](https://user-images.githubusercontent.com/39670573/192883296-969be482-be1b-47b7-a46c-4d427520dc25.png)
and when the capo changes, it looks like this:
![image](https://user-images.githubusercontent.com/39670573/192882970-00a1492d-a429-4c16-bca2-4b821d7be013.png)
I have to place the capo comment onto the [Player Name] instead of writing it as a [Tab Symbol] (or Music Symbol), which is neither elegant nor convenient in my view.

**Comment by cameronwhite** [comment]
Yeah, I think there are probably two separate features needed here : one would be to have the option of displaying the capo fret / tuning name during a player change, and then the other would be changing the capo fret for a single player during the score
For the latter, it would probably be good to check whether Guitar Pro allows this for import/export purposes
